test_util/src/builders.rs

```diff
@@ -511,33 +539,30 @@ impl TestCommandOutput {
       .expect("call .split_output() on the builder")
   }
 
+  #[track_caller]
```

Comment:
See https://blog.rust-lang.org/2020/08/27/Rust-1.46.0.html#track_caller

I just learned about this and it works way better than the `failed_position()` thing.

Reply:
Nice find